build/cmake/lib/core/CMakeLists.txt

```diff
@@ -65,6 +65,9 @@ foreach(lib JPEG PNG TIFF)
         endif()
 
         wx_lib_link_libraries(wxcore PRIVATE ${${lib}_LIBRARIES})
+        if(lib STREQUAL NANOSVG)
+            wx_lib_compile_definitions(wxbase PUBLIC wxHAS_NANOSVG)
+        endif()
```

Comment:
This is targeting the `base` library instead of `core`. But these options should not be added as compile definitions, but via `setup.h`.